js/monster.js

```diff
@@ -196,6 +196,18 @@ class Player extends Monster{
         }
         tick(); // cycling takes a turn
     }
+    Exchange(){      
+      if(this.hp > 3){
+        this.spells[this.spells.length] = shuffle(Object.keys(spells)).splice(0, numSpells);
+        this.hp-= 3;
+      }else if(this.hp == 2 && this.spells[0] == undefined){
+        this.spells[0] = shuffle(Object.keys(spells)).splice(0, numSpells);
+        this.hp--;
+      } 
+      else {
+        alert("you don't have enough life")
+      }
+    }
 }
```

Comment:
how about this?
```js
    Exchange(){
      const {hp, spells} = this
      if(hp > 3){
        spells.push(shuffle(Object.keys(spells)).splice(0, numSpells));
        hp-= 3;
      }else if(hp == 2 && spells[0] == undefined){
        spells.unshift(shuffle(Object.keys(spells)).splice(0, numSpells));
        hp--;
      } 
      else {
        alert("you don't have enough life")
      }
    }
}
```

Reply:
I think I got it, but then understand is really possible, in a world were I thinking is to be ? :thinking: